pages/learn/glossary.md

```diff
@@ -47,6 +47,16 @@ The two are, however, entirely different pieces of the conceptual puzzle, with J
 The JSON format is an open format, with its own [homepage](https://www.json.org/), and specifications published in the [ECMA-404](https://www.ecma-international.org/publications-and-standards/standards/ecma-404/) and [RFC-8259](https://datatracker.ietf.org/doc/html/rfc8259) documents from ECMA and the IETF respectively.
 In particular, it is not managed or developed by the JSON Schema team, who simply make use of the format.
 
+### JSON Hyper-Schema
+
+JSON Hyper-Schema extends JSON Schema, offering a [vocabulary](#vocabulary) to annotate JSON documents with hypermedia controls. This extension facilitates the description of links and actions that can be executed on JSON data, making it a potent tool for developing hypermedia-driven APIs.
+
+The essence of JSON Hyper-Schema lies in its ability to define links and actions that can be executed on JSON data. This is achieved through the use of the `links` keyword, which allows for the creation of dynamic, interactive data representations. For example, a JSON document representing a blog post might include an "author" property. The JSON Hyper-Schema that describes this document could include a template for a hypermedia control that uses the author's identifier in the actual data to construct a link to the author's profile. This link is a template that, when applied with the actual data, provides direct access to the author's profile, enhancing the developer experience by offering a seamless navigation experience.
+
+This approach not only simplifies the interaction with JSON data but also enhances the overall developer experience by providing direct access to related resources with a streamlined process that automates additional HTTP requests. It allows developers to create rich, interactive JSON documents that can dynamically adapt to the data they represent, making JSON Hyper-Schema a potent tool for developing hypermedia-driven APIs.
+
+To sum up, JSON Hyper-Schema enhances JSON Schema by introducing features for creating hypermedia controls. This allows for the development of more interactive APIs while preserving the integrity of existing JSON HTTP APIs. It adds a layer of interactivity to JSON documents, making it easier to interact with remote JSON resources.
```

Comment:
> This allows for the development of more interactive APIs while preserving the integrity of existing JSON HTTP APIs.

You used different words, but you still have the same problem. You're making a vague statement that you don't explain in any way. I suggest replacing the previous paragraph with an explanation of this concept.

Imagine the scenario where you have an existing API that serves JSON and you'd like to introduce some hypermedia concepts. One option is to change your API to use a hypermedia enabled media type like HAL, but that would mean changing your whole API to use a different media type. All of the consumers of your API would need to change their code to process HAL rather than plain JSON.

But, with JSON Hyper-Schema, you can add those hypermedia controls to your API without changing the JSON that you API returns. The schemas are associated with the JSON response through an HTTP Link header. Clients can construct the hypermedia controls from that schema and the JSON response, but they can also ignore the schema and use the plain JSON. That means that linking a hyper-schema to a response is a backward compatible change. Existing code consuming the API will still work the way it always has. Consumers can start using the hyper-schema that came with the response when they're ready.

This also means that the API doesn't need to be changed completely over to using hypermedia all at once. You can add hypermedia controls a little at a time.

The hard part for you is going to be writing something _brief_ that gets that concept across without going into too many details.